src/utils/backup.ts

```diff
@@ -1,7 +1,7 @@
-/* eslint-disable max-len */
-import {fs} from 'mz';
 import * as path from 'path';
 
+import {fs} from 'mz';
```

Comment:
Should remove this outdated dependency in favor in built-in node equivalent.

Reply:
Done